.kokoro/release.sh

```diff
@@ -46,15 +46,19 @@ create_settings_xml_file() {
 
 setup_environment_secrets
 
-# Pick the right package to release based on the Kokoro job name.
+# Pick the right package to release and checkout the latest release tag
```

Comment:
Is there a way we can pass the commit ID to the kokoro script? The triggering GitHub event (a GItHub Release event) that kicked off auto-release definitely contains the commit ID and/or release tag of the release.

Just picking up latest is pretty prone to race conditions.

Reply:
Good idea. Updated the script to checkout the PR that triggered the release-trigger and therefore the Kokoro job. But with this setup, if we want to manually start the kokoro job, we need to also manually add the env var.